urlwatch

```diff
@@ -370,6 +394,19 @@ if __name__ == '__main__':
             if display_errors:
                 details += foutput('error', job, (repr(error) +
                         '\n' + str(error)).strip(), summary)
+        except OSError, error:
+            log.error('got OSError while trying to execute diff command: %s' % error)
+            if display_errors:
+                details += foutput('error', job, error, summary)
+        except KeyError, error:
+            log.error('got error while attempting to replace diff command\'s placeholder(s): %s' % error)
+            if display_errors:
+                details += foutput('error', job, error, summary)
+        except subprocess.CalledProcessError, error:
+            log.error('got error while executing diff command %s: %s' % (error.cmd, error.output))
```

Comment:
Here it would be `log.error('got error while executing diff command %s: %s', error.cmd, error.output)`